pyclient/examples/import_test_data.py

```diff
@@ -0,0 +1,37 @@
+#
+#  Copyright (c) 2016-2021 Deephaven Data Labs and Patent Pending
+#
+""" Demo the steps to upload CSV data into Deephaven
+
+    1. use pyarrow to load external data of interest into memory
+    2. prepare the data
+    3. import the data into the Deephaven server
+
+"""
+
+from pyarrow import csv
+
+from examples.csv_downloader import download_csv
+from pydeephaven import Session, Table
+from pydeephaven.utils import is_deephaven_compatible
+
+
+def import_taxi_records(dh_session: Session) -> Table:
+
+    # download the CSV data and read it into a pyarrow table and prepare it for uploading into DH
+    csv_file_name = download_csv(url="https://nyc-tlc.s3.amazonaws.com/trip+data/yellow_tripdata_2020-12.csv")
```

Comment:
I am a bit confused here, the variable is called `csv_file_name`, but the name of the function that is being called is `download_csv`.  It sounds like `csv_file_name` is the csv contents?

Reply:
I really like how readable this code ends up being.   In general the argument names help a lot too, but is pretty straightforward to follow in terms of intent.